js/tests/unit/modal.js

```diff
@@ -976,4 +976,42 @@ $(function () {
         backdrop: 'static'
       })
   })
+
+  QUnit.test('should get modal-static class when clicking outside of modal-content if backdrop = static', function (assert) {
```

Comment:
this test already exist here: https://github.com/joakimriedel/bootstrap/blob/joakimriedel/defaultPrevented/js/tests/unit/modal.js#L892

You can improve it by adding your `expect`: `assert.ok($modal.hasClass('modal-static'), 'has modal-static class')`

Reply:
OK, this is done. It required a modification of the old test, adding the transition-duration style. Is that OK?

Reply:
Hmm my bad, I didn't saw the `transition-duration` so you were right it should be better to split those tests

Reply:
OK, reverted.